### PR TITLE
ibm-acf: Change location of public keys

### PR DESCRIPTION
**Pull request by human3rr**
Moves location of public keys from /etc/acf/
to /srv/ibm-acf/

### CODE REVIEW

**Review by joseph-reynolds — APPROVED** [review]
Approved with note: This change is co-req with https://github.ibm.com/openbmc/ibm-acf/pull/12

File: meta-ibm/recipes-phosphor/ibm-acf/ibm-acf.bb [hunk 1]
```diff
@@ -19,10 +19,12 @@ DEPENDS = " \
...
-                   ${D}/${sysconfdir}/acf/ibmacf-dev.key
+    install -d ${D}/${servicedir}/ibm-acf
+    install -m 755 ${S}/subprojects/ce-login/p10-celogin-lab-pub.der \
+                   ${D}/${servicedir}/ibm-acf/ibmacf-dev.key
 }
 
 
 FILES_${PN} += " ${base_libdir}/security/* \
-                 ${sysconfdir}/acf/*"
+                 ${sysconfdir}/acf/* \
```
Comment (joseph-reynolds) [inline meta-ibm/recipes-phosphor/ibm-acf/ibm-acf.bb:29]: What files are in `${sysconfdir}/acf/*`?  I believe it should be an empty directory (ready for ACF file uploads); is that right?

**Review by joseph-reynolds — APPROVED** [review]
Looks good to me.
We'll have to tell our early users when to move to the new directory.

File: meta-ibm/recipes-phosphor/ibm-acf/ibm-acf.bb (hunk 1 above)
Comment (human3rr) [reply meta-ibm/recipes-phosphor/ibm-acf/ibm-acf.bb:29]: That is correct. Maybe I don't need to include that line since we're not installing anything.

**Pull request closed without merging**